build.gradle

```diff
@@ -1,83 +1,72 @@
 plugins {
     id "com.jfrog.bintray" version "1.1"
-    id "com.github.jruby-gradle.base" version "1.5.0"
     id "java"
     id "checkstyle"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
+
 repositories {
     mavenCentral()
     jcenter()
 }
-configurations {
-    provided
-}
 
+group = "io.github.sonots"
 version = "0.3.3"
+description = "A filter plugin for Embulk to change timestamp format."
+
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.9.12"
-    provided "org.embulk:embulk-core:0.9.12"
-    compile  "io.github.medjed:JsonPathCompiler:0.1.+"
+    compileOnly  "org.embulk:embulk-core:0.9.23"
+    compile("io.github.medjed:JsonPathCompiler:0.1.+") {
```

Comment:
I do not suggest this type of version declaration: `0.1.+`, especially for Embulk plugins. Developers should be aware of exact versions of dependency librarires, which is also managed in `.lockfile`.